src/app/helptext/task-calendar/cloudsync/cloudsync-form.ts

```diff
@@ -67,6 +67,16 @@ transfer_mode_tooltip: T('<i>SYNC</i> makes files on the destination system iden
 
 transfer_mode_validation : [ Validators.required ],
 
+transfer_mode_warning_sync: T('<b>SYNC</b>: Files on the destination are <i>changed</i> to match those on the\
+ source. If a file does not exist on the source, it is also <i>deleted</i>\
```

Comment:
Avoiding the "if-then" sentence.

```suggestion
 A file that does not exist on the source is <i>deleted</i> from\
```

Reply:
Unfortunately, this is less clear.